INaturalistIOS/LoginViewController.m

```diff
@@ -442,6 +443,8 @@ - (void)configureEditableTextCell:(EditableTextFieldCell *)cell forIndexPath:(NS
     if (indexPath.item == 0) {
         placeholderText = NSLocalizedString(@"Username", @"Placeholder text for the username text field in signup");
         
+        cell.textField.rightViewMode = UITextFieldViewModeAlways;
+        cell.textField.rightView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 15.0, cell.textField.frame.size.height)];
```

Comment:
I wanted to add some padding for RTL text and didn't want to change the textfield size.